app/src/main/java/org/gdg/frisbee/android/api/GdeDirectoryClient.java

```diff
@@ -0,0 +1,46 @@
+package org.gdg.frisbee.android.api;
+
+import com.google.gson.FieldNamingPolicy;
+
+import org.gdg.frisbee.android.BuildConfig;
+import org.gdg.frisbee.android.app.App;
+import org.gdg.frisbee.android.utils.Utils;
+
+import retrofit.RestAdapter;
+import retrofit.converter.GsonConverter;
+
+/**
+ * Created by <a href="mailto:marcusandreog@gmail.com">Marcus Gabilheri</a>
+ *
+ * @author Marcus Gabilheri
+ * @version 1.0
+ * @since 7/28/15.
+ */
+public class GdeDirectoryClient {
+
+    // This variable is meant to be changed as pleased while debugging
+    private static final RestAdapter.LogLevel LOG_LEVEL = RestAdapter.LogLevel.NONE;
+    public static final String API_URL = "https://gde-map.appspot.com";
+
+    private GdeDirectoryClient() {
+    }
+
+    // We can make this a singleton to avoid instantiating this twice
+    private static GdeDirectory gdeDirectory;
```

Comment:
Yes totally. It would be really cool if we could use Dagger to inject
these. Maybe we can in the future.

On Tue, Jul 28, 2015, 9:04 PM Marcus Gabilheri <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/api/GdeDirectoryClient.java
> <https://github.com/gdg-x/frisbee/pull/432#discussion_r35679397>:
>
> > + *
> > + * @author Marcus Gabilheri
> > + * @version 1.0
> > + * @since 7/28/15.
> > + */
> > +public class GdeDirectoryClient {
> > +
> > +    // This variable is meant to be changed as pleased while debugging
> > +    private static final RestAdapter.LogLevel LOG_LEVEL = RestAdapter.LogLevel.NONE;
> > +    public static final String API_URL = "https://gde-map.appspot.com";
> > +
> > +    private GdeDirectoryClient() {
> > +    }
> > +
> > +    // We can make this a singleton to avoid instantiating this twice
> > +    private static GdeDirectory gdeDirectory;
>
> Makes sense. So just expose those methods and let the App object handle
> the management of the instances?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/432/files#r35679397>.
>